ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.15-nightly-x86.yaml

```diff
@@ -0,0 +1,53 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-azure-415
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: control-plane-ipsec-24nodes
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: Standard_D4s_v3
+      CUSTOM_AZURE_REGION: centralus
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-azure-ipi-ovn-ipsec
+  timeout: 8h0m0s
```

Comment:
Since this is 24 Nodes, the timeout needs to be updated to `5h0m0s`
to line it up with AWS https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml#L38-L49

Reply:
@krishvoor Thanks for the comment, I will include this change in a new pr.

Reply:
Thanks for your response @qiliRedHat!

Reply:
Change in pr https://github.com/openshift/release/pull/46293